frontend/src/screens/studyBanner.tsx

```diff
@@ -0,0 +1,136 @@
+import { useSearchStudies, useParticipantStudies } from './learner/studies';
+import { ParticipantStudy } from '@api';
+
+interface BannerSectionProps {
+    title: string;
+    mainText: string;
+    subText?: string;
+    value?: string | number;
+    hasValue: boolean;
+    onClick?: () => void;
+}
+
+const BannerSection: React.FC<BannerSectionProps> = ({ title, mainText, subText, value, hasValue, onClick }) => {
+    return (
+        <Box p="xl" style={{ flex: 1, minWidth: '200px', maxWidth: '300px' }} fz='lg'>
+            <Title order={3} size='1rem' style={{ textAlign: hasValue ? 'center' : 'left' }}>
+                {title}
+            </Title>
+            {hasValue ? (
+                <Center style={{ height: '80px', marginBottom: '10px' }}>
+                    <Text color="purple" size="70px" style={{ textAlign: 'center', fontWeight: 200 }}>
+                        {value}
+                    </Text>
+                </Center>
+            ) : (
+                <>
+                    <Text mt="md" size="sm" style={{ textAlign: 'start' }}>
+                        {mainText}
+                    </Text>
+                    {subText && (
+                        <Text size="sm" style={{ textAlign: 'start' }}>
+                            {subText}
+                        </Text>
+                    )}
+                    {onClick && (
+                        <Anchor color="blue" onClick={onClick} style={{ display: 'flex', alignItems: 'center', marginTop: '5px' }}>
+                            <Text size="sm" style={{ fontSize: '14px', fontWeight: 700 }}> Start your first study </Text>
+                            <IconArrowRight style={{ marginLeft: '5px' }} />
+                        </Anchor>
+                    )}
+                </>
+            )}
+        </Box>
+    );
+};
+
+const StudyBanner: React.FC = () => {
+    const { filteredStudies } = useSearchStudies();
+    const { studies = [] } = useParticipantStudies() as { studies: ParticipantStudy[] };
+    const navigate = useNavigate();
+
+    const [totalCompletedCount, setTotalCompletedCount] = useState<number>(0);
+    const [badgesEarned, setBadgesEarned] = useState<number>(0);
+    const [totalPointsEarned, setTotalPointsEarned] = useState<number>(0);
+    const [hasCompletedStudies, setHasCompletedStudies] = useState<boolean>(false);
+    const [fiveMinuteStudies, setFiveMinuteStudies] = useState<ParticipantStudy[]>([]);
+    const [dataFetched, setDataFetched] = useState<boolean>(false);
+
+    useEffect(() => {
+        if (studies.length > 0 || filteredStudies.length > 0) {
+            setTotalCompletedCount(studies.reduce((sum, study) => sum + (study.completedCount || 0), 0));
+            setBadgesEarned(studies.reduce((count, study) => count + (study.learningPath?.completed ? 1 : 0), 0));
+            setTotalPointsEarned(studies.reduce((sum, study) => sum + (study.learningPath?.completed ? (study.totalPoints || 0) : 0), 0));
```

Comment:
Incorrect logic, points should be calculated regardless of whether the learning path is completed